Corona-Warn-App/src/main/res/layout/fragment_statistics_explanation.xml

```diff
@@ -217,7 +272,7 @@
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
                     android:layout_marginStart="@dimen/guideline_start"
-                    android:layout_marginTop="36dp"
+                    android:layout_marginTop="32dp"
```

Comment:
Could you use dimension values such as `spacing_medium`/`spacing_normal`/etc. when you already updating values?

Reply:
afaik last year we decided not to be hardly attached to dimens.xml in xml layouts